apps/shell/src/app/modules/prompt/snippets/snippets.tsx

```diff
@@ -0,0 +1,129 @@
+import { Button, Modal } from '@cased/ui';
+import { useMemo, useState } from 'react';
+import SnippetCategory from './category';
+import SnippetListItem from './list-item';
+
+function SnippetsDetail() {
+  return (
+    <div className="h-full w-full border-t border-zinc-700 bg-zinc-900 p-4">
+      <div className=" mb-2 font-semibold text-white">Content Name</div>
+      <div className="mb-4 border-b border-zinc-600 pb-2 text-sm text-zinc-500">
+        content description
+      </div>
+      <div className="mb-2 space-y-1">
+        <h4 className="text-sm text-zinc-400">Arguments</h4>
+        <div>
+          <label htmlFor="label1" className="mb-1 mr-2 text-sm">
+            label1
+          </label>
+          <input
+            type="text"
+            className="border border-zinc-600 bg-transparent text-sm focus:bg-transparent focus:outline-none"
+            value="input1"
+            onChange={() => {}}
+          />
```

Comment:
There's actually a much easier way to structure labels (see below). Also try to think of the label name in two ways.

* Screen readers will hear this label name for anybody with a disability who is trying to find a specific input
* The name will help greatly with E2Es and testing. As the label can be used as a pseudo ID to find an input

That said, here are some proposed tweaks below that could help with usability on this. Not sure this is 100% correct, but worth considering.

```suggestion
          <label>
            {/* adding sr-only will hide the text safely, but still make it usable for tests and screen readers */}
            <span className="mb-1 mr-2 text-sm">Argument 1</span>
            <input
              type="text"
              className="border border-zinc-600 bg-transparent text-sm focus:bg-transparent focus:outline-none"
              value="input1"
              onChange={() => {}}
            />
          </label>
```